lib/realtime_web/router.ex

```diff
@@ -131,8 +131,10 @@ defmodule RealtimeWeb.Router do
 
   defp check_auth(conn, secret_key) do
     secret = Application.fetch_env!(:realtime, secret_key)
+    blocklist = Application.get_env(:realtime, :block_list, [])
```

Comment:
@filipecabaco you named it `blocklist` in runtime.exs: https://github.com/supabase/realtime/pull/1069/files#diff-e43c5cbf91db7a8062b6cb860cbf118296c1b4c7ee32fdcf702e54234ba38092R56

```suggestion
    blocklist = Application.get_env(:realtime, :blocklist, [])
```